common/src/main/resources/messages/messages.properties

```diff
@@ -214,6 +214,7 @@ cluster.ccm.upgrade.finished=Cluster Connectivity Manager upgrade finished.
 cluster.kerberosconfig.validation.failed=Kerberos config validation failed with: {0}
 
 cluster.cloudconfig.validation.failed=Cloud config validation failed with the following errors: {0}
+cluster.provider.validation.warning=Some warnings were found during cloud platform validation: {0}
```

Comment:
I would reformulate to match the above error message:
"Cloud platform validation succeeded with the following warnings: {0}"